test/unit/instance/changed.test.js

```diff
@@ -31,14 +31,17 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
 
     it('should return falsy for unchanged primitive', function () {
       var user = this.User.build({
-        name: 'a'
+        name: 'a',
+        meta: null
       }, {
         isNewRecord: false,
         raw: true
       });
 
       user.set('name', 'a');
+      user.set('meta', null);
       expect(user.changed('name')).to.equal(false);
+      expect(user.changed('meta')).to.equal(false);
     });
```

Comment:
We might want a test for checking that `null` does return in changed if it was a non-null value before.